src/pages/components/dropdown/accessibility.mdx

```diff
@@ -6,98 +6,129 @@ description:
+
+### Keyboard interactions
+
+A dropdown component and its variants multiselect and combo box are reached by
+`Tab`, with navigation of the options by `Up` and `Down` arrow keys. However,
+the keys for opening the component and selecting its options are different for
+each variant.
+
+<Row>
+<Column colLg={8}>
+
+![Tab to dropdown then operate with arrow keys](images/dropdown-accessibility-1.png)
+
+<Caption>
+
+Dropdowns are in the tab order and operable by Up and Down arrow keys, once
+opened.
+
+</Caption>
+
+</Column>
+</Row>
+
+#### Dropdown
+
+The default dropdown opens with `Enter`, `Space` or `Down arrow`, with focus
+moving to the currently selected option. Arrow keys change options, and any
+option with focus is selected with `Space` or `Enter`, which also closes the
+dropdown. Pressing `Esc` or `Tab` closes a dropdown without changing the
+selected option.
+
+<Row>
+<Column colLg={8}>
+
+![Dropdowns are activated and their options selected with Space or Enter keys. Esc closes without changing option](images/dropdown-accessibility-2.png)
+
+<Caption>
+  The Space and Enter keys can be used to open dropdowns and also select an
+  option with focus. Pressing Esc cancels and closes the interaction.
+</Caption>
+
+</Column>
+</Row>
+
+#### Multiselect
+
+The multiselect opens with the `Enter` or `Space` keys, which are also used to
+toggle the selection of any option. Once items are selected, a tag appears in
+the field, showing a numerical representation of the number of items selected
+along with an ‘x’. Pressing `Esc` closes the multiselect. Pressing `Delete`
+while focus is in the collapsed field will clear all selected options.
 
 <Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListHead>
-        <StructuredListRow head>
-          <StructuredListCell head>
-            Automated test environment
-          </StructuredListCell>
-          <StructuredListCell head>Results</StructuredListCell>
-        </StructuredListRow>
-      </StructuredListHead>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - macOS Mojave version 10.14.2 with VoiceOver
-            <br />
-            - Chrome version 77.0.3865.90
-            <br />
-            - Dynamic Assessment Plugin (DAP) version 1.8.0.0 - IBM
-            Accessibility WCAG 2.1 Sept. 2019 Ruleset
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>DAP test:</strong>
-            <br />- Violations
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![Space and Enter keys open multiselects and toggle the selection of checkbox options. Esc closes.](images/dropdown-accessibility-3.png)
+
+<Caption>
+  Space and Enter keys open multiselects and toggle the state of the checkboxes
+  in the list. Esc closes the list.
```

Comment:
A small thing, but adding "Pressing Esc closes the list" would be consistent with the caption before this.

Reply:
We don't normally use verbs in the captions for these things (since, as you note, we tend to in the surrounding text). It's hardly a fast rule, but given the visual and the preamble, IMO, it's not needed. I do note that one of the captions on the page had the word "press" once. I've removed it for consistency.